CefSharp.Core/Internals/ClientAdapter.cpp

```diff
@@ -290,6 +307,28 @@ namespace CefSharp
             }			
         }
 
+        void ClientAdapter::OnResourceRedirect(CefRefPtr<CefBrowser> browser, CefRefPtr<CefFrame> frame, const CefString& oldUrl, CefString& newUrl)
+        {
+            IRequestHandler^ handler = _browserControl->RequestHandler;
+            if (handler != nullptr)
+            {
+                auto managedNewUrl = StringUtils::ToClr(newUrl);
+                handler->OnResourceRedirect(_browserControl, frame->IsMain(), StringUtils::ToClr(oldUrl), managedNewUrl);
+
+                //TODO: Determine if there is a better way to pass the reference in rather than reassigning the string
+                newUrl = StringUtils::ToNative(managedNewUrl);
```

Comment:
No such thing, unfortunately. There are tricks you can do if you're going the other way, or if you want to treat a `String` as an `IStream`, but nothing to help out here so far as I know. :frowning: